app/controllers/users_controller.rb

```diff
@@ -0,0 +1,33 @@
+class UsersController < ApplicationController
+  before_action :authenticate_user!
+
+  def index;end
+
+  def show
+    @user = User.find(params[:id])
+    render "user/show"
+  end
+
+  def edit
+    @user = current_user
+    render "user/edit"
+  end
+
+  def update_password
+    @user = current_user
```

Comment:
Why do we needd to use `@user` variable, we can use current_user directly in views?